### PR TITLE
Fixes grenade belt and bandolier UI obscuring half the screen

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl: 
fix: Grenade belts and bandoliers will no longer obscure half the screen when they're completely filled.
/:cl:

Also fixes storage size of the grenade belts so you can put stuff back in if you take it out

### CODE REVIEW

**Comment by Supermichael777** [comment]
pics

**Comment by Bawhoppen** [comment]
before:
dreamseeker_2017-06-19_18-50-00
after:
dreamseeker_2017-06-12_04-11-53

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -289,7 +289,9 @@
 	icon_state = "grenadebeltnew"
 	item_state = "security"
 	max_w_class = WEIGHT_CLASS_BULKY
+	display_contents_with_number = 1
```
Comment (Cyberboss) [inline code/game/objects/items/weapons/storage/belt.dm]: TRUE

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -374,6 +376,7 @@
 	icon_state = "bandolier"
 	item_state = "bandolier"
 	storage_slots = 18
+	display_contents_with_number = 1
```
Comment (Cyberboss) [inline code/game/objects/items/weapons/storage/belt.dm]: TRUE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**